src/astro/sql/operators/load_file.py

```diff
@@ -82,13 +85,23 @@ def load_data_to_table(self, input_file: File) -> Table:
             input_file.conn_id,
             normalize_config=self.normalize_config,
         ):
-            database.load_pandas_dataframe_to_table(
-                source_dataframe=file.export_to_dataframe(),
-                target_table=self.output_table,
-                if_exists=if_exists,
-                chunk_size=self.chunk_size,
-            )
-            if_exists = "append"
+            if self.use_native_support and database.check_optimised_path(
+                source_file=file, target_table=self.output_table
+            ):
+                database.optimised_transfer(
+                    source_file=file,
+                    target_table=self.output_table,
+                    if_exists=self.if_exists,
+                    **self.kwargs,
+                )
+            else:
+                database.load_pandas_dataframe_to_table(
+                    source_dataframe=file.export_to_dataframe(),
+                    target_table=self.output_table,
+                    if_exists=if_exists,
+                    chunk_size=self.chunk_size,
+                )
+                if_exists = "append"
```

Comment:
Why are we setting `if_exists` to append here?

Reply:
@tatiana we are iterating over files and the data needs to be appended in the same table.

Reply:
The operator should not have to worry about iterating through files - this should be an internal implementation. from within the Database. Also, it is probably worth adding a comment saying this line is used when we have multiple files.

Reply:
Done.